packages/profiler/src/index.ts

```diff
@@ -61,7 +62,7 @@ export class Profiler {
       //be resolved, it will show up as a compile error rather than a Truffle
       //error.
       try {
-        return await resolver.resolve(filePath, importedFrom);
+        return await resolver.resolve(filePath, importedFrom, { compiler });
```

Comment:
would like to see a debug-log in the exception path. Also, are the comments wrt structured errors up to date?

Reply:
(I believe the comments are up-to-date, but which comment are you asking about?)

Reply:
added debug statement